src/plone/restapi/services/workflow/transition.py

```diff
@@ -77,20 +90,40 @@ def reply(self):
             self.request.response.setStatus(400)
             return dict(error=dict(type="Bad Request", message=str(e)))
 
-        history = self.wftool.getInfoFor(self.context, "review_history")
-        action = history[-1]
-        if six.PY2:
-            action["title"] = self.context.translate(
-                self.wftool.getTitleForStateOnType(
-                    action["review_state"], self.context.portal_type
-                ).decode("utf8")
+        sm = getSecurityManager()
```

Comment:
I would use a context manager here

Reply:
Ok, i have amended a context manager in my last commit and moved the whole security manager management into it.